db/migrate/20230809083044_add_all_conditions_met_to_project.rb

```diff
@@ -0,0 +1,39 @@
+class AddAllConditionsMetToProject < ActiveRecord::Migration[7.0]
+  def up
+    add_column :projects, :all_conditions_met, :boolean
+
+    if update_all_projects
+      remove_column :conversion_tasks_data, :conditions_met_confirm_all_conditions_met, :boolean
+    end
+  end
+
+  def down
+    add_column :conversion_tasks_data, :conditions_met_confirm_all_conditions_met, :boolean
+
+    if revert_all_projects
+      remove_column :projects, :all_conditions_met, :boolean
+    end
+  end
+
+  def update_all_projects
+    Conversion::Project.transaction do
+      Conversion::Project.all.each do |project|
+        tasks_data = project.tasks_data
+        all_conditions_met = tasks_data.conditions_met_confirm_all_conditions_met
+        project.all_conditions_met = all_conditions_met
+        project.save(validate: false)
+      end
+    end
+  end
+
+  def revert_all_projects
```

Comment:
Once we are confident the up works, we might re-think the down.

If the up has worked, and it kind of has to now, we know that project.all_conditions_met will be either true or false, so can we skip the transactions and checks here?

Is it something like (untested - just food for thought):

```
def revert_all_projects
  Conversion::Project.all.each do |project|
        all_conditions_met = project.all_conditions_met || false
        project.update_attribute(:all_conditions_met, all_conditions_met)
      end 
end
```